07a_system_activity_plots.R

```diff
@@ -189,75 +190,139 @@ renderSystemPlot <- function(id) {
         '#1b8297'
       )
          df <- system_activity_prep_detail()
+         mid_plot <- 4.5
        }
        
     s <- max(df$yend) + 20
     num_segments <- 20
     segment_size <- get_segment_size(s/num_segments)
-
+    
+    total_clients <- df %>%
+      filter(InflowOutflow == "Inflow") %>%
+      pull(values) %>%
+      sum()
+    
+    inflow_to_outflow <- df %>%
+      filter(Status %in% c("Inflow", "Outflow")) %>%
+      pull(values) %>%
+      sum()
+    
+    if(total_clients < 11){
```

Comment:
I would use `validate()` instead, to be more intuitive and consistent with how we validate no data in our charts and tables. Something like:

```
validate(
      need(
        total_clients >= 10,
        message = "The data has been filtered to less than 11 records, therefore\n
          the plot will not be displayed for privacy purposes."
      )
    )
```

Reply:
Nice- done, thank you!